src/map/atcommand.c

```diff
@@ -1714,6 +1714,15 @@ ACMD(bodystyle)
 
 	memset(atcmd_output, '\0', sizeof(atcmd_output));
 
+	if (!((sd->job&MAPID_THIRDMASK) == MAPID_GUILLOTINE_CROSS || (sd->job&MAPID_THIRDMASK) == MAPID_GENETIC
```

Comment:
@Smokexyz you can't replace this with just one bitmask (see the definitions for the MAPID values lower than MAPID_1_1_MAX: they aren't bitmask-compatible)

This needs to be done similar to how @Jedzkie originally did. If you want to get rid of the `!`, you can do the following:

```
if ((sd->job & MAPID_THIRDMASK) != MAPID_GUILLOTINE_CROSS
 && (sd->job & MAPID_THIRDMASK) != MAPID_GENETIC
 && (sd->job & MAPID_THIRDMASK) != MAPID_MECHANIC
 && (sd->job & MAPID_THIRDMASK) != MAPID_ROYAL_GUARD
 && (sd->job & MAPID_THIRDMASK) != MAPID_ARCH_BISHOP
 && (sd->job & MAPID_THIRDMASK) != MAPID_RANGER
 && (sd->job & MAPID_THIRDMASK) != MAPID_WARLOCK
 && (sd->job & MAPID_THIRDMASK) != MAPID_SHADOW_CHASER
 && (sd->job & MAPID_THIRDMASK) != MAPID_MINSTRELWANDERER
) {
	clif->message(fd, msg_txt(35));	// This job has no alternate body styles.
	return false;
}
```